haskell-ide-engine.cabal

```diff
@@ -98,6 +96,9 @@ library
                      , vector
                      , versions
                      , yaml >= 0.8.31
+                     , hie-bios
+                     , bytestring-trie
+                     , unliftio
```

Comment:
unliftio dependency is unused I think? It was intended to replace monad-control as the monad stack is now a RIO.

Reply:
Need to remove the `unliftio` dependency.